kbchat/kbchat_test.go

```diff
@@ -0,0 +1,119 @@
+package kbchat
+
+import (
+	"bytes"
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
```

Comment:
Any specific reason you're importing v2 from gopkg instead of importing straight from github?

Reply:
That's what the [package's readme says to do](https://github.com/go-yaml/yaml) in the installation instructions.